snuba/datasets/events.py

```diff
@@ -242,6 +243,21 @@ def __init__(self) -> None:
             ]
         )
 
+        promoted_tags_mapping = {
+            col.flattened: col.flattened for col in promoted_tag_columns
+        }
+        promtoed_tags_context_mapping = {
+            col.flattened.replace("_", "."): col.flattened
+            for col in promoted_context_tag_columns
+        }
+        promoted_tags_mapping.update(promtoed_tags_context_mapping)
+        promoted_columns_spec = {
+            "tags": PromotedColumnSpec(promoted_tags_mapping),
+            "contexts": PromotedColumnSpec(
+                {col.flattened: col.flattened for col in promoted_context_columns}
+            ),
+        }
```

Comment:
This is horrible but that's the way the mapping works today. Anyway we are getting rid of this dataset.